src/components/Personalization/flicker/index.js

```diff
@@ -10,8 +10,15 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { createNode, appendNode, removeNode } from "../../../utils/dom";
+import {
+  createNode,
+  appendNode,
+  removeNode,
+  findById
+} from "../../../utils/dom";
 
+const CONTAINERS_HIDING_ID = "alloy-prehiding";
```

Comment:
Shouldn't this be configurable? With two alloy instances, you may want to have two different hiding ids.

Reply:
Added `prehidingId` so we can have multiple instances with their own configurations.